### ISSUE TITLE
Run shmem_perf_suite to check for regression against v1.3.5

### ISSUE THREAD

**Issue by davidozog** [Sandia-OpenSHMEM/SOS#664]

**Comment by davidozog** [comment]
Things look great so far on this front.  I've A/B tested v1.3.5 against master https://github.com/Sandia-OpenSHMEM/SOS/commit/0b3b75ce8a31960a10a62207ada2b956fb795a23 using libfabric v1.6.0 and libfabric v1.5.x.  This includes the unidirectional bw put/get and AMO latency tests in `shmem_perf_suite.`  No regressions on PSM2.

Working on Cray Aries...

**Comment by davidozog** [comment]
Things still look very good on this issue.   I see no regressions in shmem_perf_suite or GUPS. (I also see a decent speedup in GUPS with `atomic_xor` over SOS v1.3.5's get/put version on PSM2 and Aries).  In the shmem_perf_suite, I've checked the blocking/non-blocking bandwidth for put/get and atomic latency tests on PSM2 and GNI.

Side note 1:  One tough lesson I've learned here is to never forget to consider process affinity with the `bind-to "X"` argument to the launcher (already described in our performance tuning wiki page).  The optimal value for `"X"` depends on the number of processes, threads, NUMA effects, and the application.  Binding to `core` is usually a good place to start.  Interestingly, the effect of *not* binding processes on PSM2 seems to have considerably improved from v1.5.x and v1.6.0 of libfabric, but I haven't figured out why.

Side note 2:  For some reason SOS v1.3.5 + libfabric v1.6.0 fetching AMO latency looks really bad (~80 microseconds), but the problem is not there in SOS v1.4.1 (~6 us).  Not sure what's going on there, but it's something to keep in mind.
